src/Gizmos/axisDragGizmo.ts

```diff
@@ -126,29 +127,41 @@ export class AxisDragGizmo extends Gizmo {
                 // if the node has parent, the local transform properties (position, rotation, scale)
                 // will be recomputed in _matrixChanged function
 
+                let matrixChanged: boolean = false;
                 // Snapping logic
                 if (this.snapDistance == 0) {
-                    if ((this.attachedNode as any).position) { // Required for nodes like lights
-                        (this.attachedNode as any).position.addInPlaceFromFloats(event.delta.x, event.delta.y, event.delta.z);
+                    this.attachedNode.getWorldMatrix().getTranslation().addToRef(event.delta, tmpVector2);

```

Comment:
```suggestion
                    this.attachedNode.getWorldMatrix().getTranslationToRef(tmpVector2).addInPlace(event.delta);
```

Reply:
getTranslationToRef returns a matrix and not a vector3